vali_objects/position.py

```diff
@@ -2,7 +2,7 @@
 import logging
 from copy import deepcopy
 from typing import Optional, List
-from pydantic import model_validator, BaseModel, Field
+from pydantic import model_validator, BaseModel, Field, field_validator
```

Comment:
this looks like legacy from my blotched attempt, doesn't look like the file uses field_validator